### PR TITLE
Bug 1468606 - add servicesManifest method

### PR DESCRIPTION
**Pull request by petemoore**
See [bug 1468606](https://bugzil.la/1468606) for more context.

I've added a new method that returns the `manifest.json` file for a given taskcluster root url.

At the same time I added some green ticks to the go tests, since I like the ones that `npm test` produces. 😉 

Thanks!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -34,6 +34,9 @@ jspm_packages
 # Optional npm cache directory
 .npm
 
+# npm package lock file
+package-lock.json
```
Comment (djmitche) [inline .gitignore]: This doesn't hurt anything, but FYI we now use yarn instead of npm to manage packages.

**Comment by djmitche** [comment]
Looks like gometalinter needs a fix-up, but once that's fixed, good to land.

**Comment by petemoore** [comment]
Thanks guys!

**Comment by djmitche** [comment]
v10.1.0

**Pull request merged**